README.md

```diff
@@ -57,6 +57,36 @@ Historic data snapshots are also hosted on https://data.visitdata.org/
 For example, you can retrieve
 https://data.visitdata.org/processed/vendor/foursquare/asof/20200403-v0/taxonomy.json
 
+# Weather Data
+
+Weather data is retrieved from weatherapi.com. After retrieving the data, it creates a file with the following structure:
+`<state>.json:`
+```
+{
+    <county-1>: {
+        forecast: {
+            day-1-timestamp: { <weather_data> },
+            day-2-timestamp: { <weather_data> },
+            ...
+        }
+    },
+    <county-2>: { .... }
+    ...
+}
+```
+
+In the **production environment**, a task will be exececuted daily (`etl/dags/extract_weather_data.py`) to store
+the weather data into a google cloud bucket.
+ 
+ `BUCKET_NAME` variable must be defined in the airflow server
+ 
+Once the data is stored in the bucket, we can get the weather data for one state in the route 
+`weather/<state>`
```

Comment:
We might want to consolidate all of this under `https://visitdata.org/data/` and present it as a single bundle that all goes together.

Reply:
@markroth8 the only issue is that it would increase the size of the initial data load for each page before it renders. If we assume that in most cases, people will be looking at the visits and not the weather, it would be better to load it only upon request. 
In general if we plan to add more data sources which might be only displayed based on user's choices in the UI, the architecture of separating those data sources might be better?